File: 0001_0100/0002/0002.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int a;

	fscanf(f, "%d", &a);
	fprintf(q, "%d", a > 0 ? (1 + a) * a / 2 : (1 - a) * a / 2 + 1);
	return 0;
}

File: 0001_0100/0002/0002.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var n int
	fmt.Fscan(f, &n)
	if n > 0 {
		n = (1 + n) * n / 2
	} else {
		n = (1-n)*n/2 + 1
	}
	fmt.Fprint(q, n)
}
